src/routes/landing/components/CitySelector.js

```diff
@@ -76,10 +122,45 @@ class CitySelector extends React.PureComponent<PropsType> {
     }, [])
   }
 
+  renderListByLocation (): React.Node {
+    if (this.props.currentLatitude === null || this.props.currentLongitude === null) {
+      return null
+    }
+    const currentLatitude = this.props.currentLatitude
+    const currentLongitude = this.props.currentLongitude
+    let cities = this.props.cities.filter(_city => _city.live)
+      .sort((a: CityModel, b: CityModel) => compareDistance(a, b, currentLongitude, currentLatitude))
+    const numberOfClosestCities = 3
+    cities = cities.slice(0, numberOfClosestCities)
+    const maximalDistance = 90
+    cities = cities.filter(_city => currentDistance(_city, currentLongitude, currentLatitude) < maximalDistance)
+    if (cities && cities.length) {
+      return <View>
+        <CityGroup theme={this.props.theme}> Close </CityGroup>
```

Comment:
This needs to be added to the locales.json. So you have to the the translate() function and define a key in the locales.json

Reply:
Also I think Nearby sounds better. What do the others think? @markl @klinzo @NKlug @Simple-codinger